server/controllers/authController/authController.test.js

```diff
@@ -1,10 +1,14 @@
+import passport from 'passport'
 import AuthController from '.'
 import {
+  createRequest,
   createRequestWithUser,
   createResponse,
   createUser,
 } from '../../../test/fixtures'
 
+jest.mock('passport')
```

Comment:
Why do we have to mock passport? I thought it was pure JS functions, no outbound network calls or anything like that.

Reply:
[9da7fb5](https://github.com/AMP-SCZ/dpdash/pull/509/commits/9da7fb5e622ebd751b1c9c1f54f635092b59309c)
good point